recipes-devtools/ruby_%.bbappend

```diff
@@ -0,0 +1 @@
+EXTRA_OECONF_remove = "--disable-rpath"
```

Comment:
potential issue: Please add an 'SPDX-License-Identifier: MIT' comment to the file